src/Serilog.Settings.Configuration/Settings/Configuration/Assemblies/AutoAssemblyFinder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
+
+namespace Serilog.Settings.Configuration.Assemblies;
+
+class AutoAssemblyFinder : AssemblyFinder
```

Comment:
What does _Auto_ mean here? It looks more like _Composite_.

Reply:
Absolutely, I named it `AutoAssemblyFinder` because it's used when using `AssemblyFinder.Auto()`. But `CompositeAssemblyFinder` describes its purpose better. Addressed in ceb3df3fde3dc777e1f5910032ca267ef76847c2